src/server/routes/entity/publisher.js

```diff
@@ -58,6 +58,16 @@ function _setPublisherTitle(res) {
 	);
 }
 
+/**
+ * @param {object} a The first object.
+ * @param {object} b The second object.
+ * @returns {number} The sum of the two numbers.
```

Comment:
Nope - this is the difference. Also, you haven't mentioned the numbers earlier in the comment. Please format like this:

    /**
     * description goes here
     *
     * @param ...
     * @param ...
     * @returns ...
     **/

Reply:
Also, might be worth mentioning that the objects being passed in represent editions - something like "@param a The first edition to compare."